sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -40,4 +40,15 @@ private object CosmosConstants {
   object SystemProperties {
     val LineSeparator = System.getProperty("line.separator")
   }
+
+  object TableProperties {
+    val PartitionKeyDefinition = "CosmosPartitionKeyDefinition"
+    val PartitionCountSnapshot = "CosmosPartitionCountSnapshot"
+    val LastModified = "LastModifiedSnapshot"
+    val ProvisionedThroughputSnapshot = "ProvisionedThroughputSnapshot"
```

Comment:
what does the suffix Snapshot indicate here and for the other constants?

Reply:
Those properties are mutable - so the value shown in TBLPROPERTIES are the values whenever the Table was loaded. Suffix Snapshot as indication that this value can change (even rapidly) like with RU